src/CMakeLists.txt

```diff
@@ -75,4 +75,6 @@ set(tfs_SRC
 	${CMAKE_CURRENT_LIST_DIR}/weapons.cpp
 	${CMAKE_CURRENT_LIST_DIR}/wildcardtree.cpp
 	PARENT_SCOPE)
+file(GLOB_RECURSE tfs_HDR *.h)
```

Comment:
We use the glob only for clang-format (and clang-tidy in the future). Using it for the build is a bad idea (screws up cmake's change detection system AFAIK).